tests/e2e/e2e_test.go

```diff
@@ -25,51 +25,28 @@ import (
 var t = strconv.FormatInt(time.Now().Unix(), 10)
 var projName = fmt.Sprintf("ocdev-%s", t)
 
-func runCmd(cmdS string) string {
+func runCmd(cmdS string) (string, error) {
+	cmd := exec.Command("/bin/sh", "-c", cmdS)
+	session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
 
-	var cmd *exec.Cmd
-	var out, stderr bytes.Buffer
+	// wait for the command execution to complete
+	<-session.Exited
+	Expect(session.ExitCode()).To(Equal(0))
 
-	cmd = exec.Command("/bin/sh", "-c", cmdS)
-	cmd.Stdout = &out
-	cmd.Stderr = &stderr
-
-	err := cmd.Run()
-
-	if err != nil {
-		fmt.Println(fmt.Sprint(err) + ": " + stderr.String())
-		os.Exit(1)
-	}
-
-	fmt.Printf("$ %s\n%s\n\n", cmdS, string(out.Bytes()))
-	return string(out.Bytes())
+	return string(session.Out.Contents()), err
 }
 
-func createApp(appName string) {
-	runCmd("ocdev application create " + appName)
-}
+// Function get commands as oc admin
+func runOcAdmin(cmd string) (string, error) {
+	_, err1 := runCmd("oc login -u system:admin > /dev/null")
+	Expect(err1).NotTo(HaveOccurred())
 
-func getProj() string {
-	proj := runCmd("ocdev project get --short")
-	return strings.TrimSpace(proj)
-}
+	cmdOut, err2 := runCmd(cmd)
```

Comment:
would it make sense to also check err2 (`Expect(err2).NotTo(HaveOccurred`) here?
So we don't have to think about checking it after every runOcAdmin call.